tests/gtests/sw_maintest.cc

```diff
@@ -1,4 +1,4 @@
-#include "gtest/gtest.h"
+#include <gmock/gmock.h>

```

Comment:
We'll need to update `makefile` so that compilation can find "include/googletest/googlemock/"